gamemodes/terrortown/entities/weapons/weapon_tttbase.lua

```diff
@@ -360,8 +360,9 @@ if CLIENT then
 
 		-- draw crosshair dot
 		if cvCrosshairMode:GetInt() == CROSSHAIR_MODE_DOT_AND_LINES or cvCrosshairMode:GetInt() == CROSSHAIR_MODE_DOT_ONLY then
-			local xDot = mathFloor(xCenter - thicknessLine * 0.5)
-			local yDot = mathFloor(yCenter - thicknessLine * 0.5)
+			local xDot = xCenter - mathFloor(thicknessLine * 0.5)
+			local yDot = yCenter - mathFloor(thicknessLine * 0.5)
```

Comment:
Doesnt that mean a thicknessLine of 1 is invisible?

Reply:
this is just for the dot's position, if the thicknessLine is 1, it means there's no offset